doc/getting_going.rst

```diff
@@ -122,6 +122,35 @@ latest version simply do:
 
    > pip install fparser --upgrade
 
+git submodule
```

Comment:
I'm afraid I'm going to be a pain and ask for a bit of a restructure of getting_going.rst. This change is now allowing users to access master if they want the latest version of PSyclone and the documentation for users does not reflect this, it is a bit of a muddle. I think getting going should have info on getting the release version of PSyclone first then have a section explaining how to get the latest version if needed.

There should then be a reference back to this section in the developers guide. I think the developers guide needs a new section at the beginning which talks about getting going, which includes working practices and obviously includes how to get the latest version (referencing the information on how to get  fparser when cloning psyclone and how to build the two.). This section can be mostly a placeholder as long as there are appropriate words and a reference to the users section explaining how to clone and install from master.

Actually it might make more sense to have the working from master words in the developers section and then reference that from the users section. I don't really mind.